### PR TITLE
feat: Incentives for sFAANG

### PR DESCRIPTION
**Pull request by JChiaramonte7**
- Support for sFAANG incentives
- Refactored balancer incentives code to make more re-usable

### CODE REVIEW

File: components/Currency/CurrencyIcon/CurrencyIcon.tsx [hunk 1]
```diff
@@ -72,6 +72,11 @@ import iTRXIcon from '@synthetixio/assets/synths/iTRX.svg';
 import iDASHIcon from '@synthetixio/assets/synths/iDASH.svg';
 import iETCIcon from '@synthetixio/assets/synths/iETC.svg';
 import sTSLAIcon from '@synthetixio/assets/synths/sTSLA.svg';
+import sFBIcon from '@synthetixio/assets/synths/sFB.svg';
```
Comment (justinjmoses) [inline components/Currency/CurrencyIcon/CurrencyIcon.tsx:75]: Surely we don't need to do this every time. There's gotta be a better way without explicitly listing the synths here. cc @andytcf

File: constants/currency.ts [hunk 2]
```diff
@@ -46,6 +46,11 @@ export enum Synths {
 	iCEX = 'iCEX',
 	iDEFI = 'iDEFI',
 	sTSLA = 'sTSLA',
+	sFB = 'sFB',
```
Comment (justinjmoses) [inline constants/currency.ts:49]: what?

**Review by justinjmoses — APPROVED** [review]

File: queries/liquidityPools/helper.ts [hunk 3]
```diff
@@ -56,6 +56,31 @@ export async function getsTSLABalancerPool(): Promise<number> {
...
+				where: {
+					id: `\\"${poolTokenAddress.toLowerCase()}\\"`,
+				},
+			},
+			properties: ['id', 'liquidity', 'totalShares'],
+		},
+		max: 1,
+		// @ts-ignore
+	})
+		.then((result: any) => {
```
Comment (vbstreetz) [inline queries/liquidityPools/helper.ts:76]: probably complete the async/await syntax?

**Review by vbstreetz — APPROVED** [review]
Looking good 👍🏽

File: queries/liquidityPools/helper.ts [hunk 4]
```diff
@@ -56,6 +56,31 @@ export async function getsTSLABalancerPool(): Promise<number> {
 		});
 }
 
+export async function getBalancerPool(poolTokenAddress: string): Promise<number> {
```
Comment (dvd-schwrtz) [inline queries/liquidityPools/helper.ts:59]: we will move this to the new data library soon but its good for now

File: queries/liquidityPools/useBalancerPoolQuery.ts [hunk 5]
```diff
@@ -26,44 +36,43 @@ const useTSLAPoolQuery = (options?: QueryConfig<LiquidityPoolData>) => {
 	const isMainnet = useRecoilValue(isMainnetState);
 
 	return useQuery<LiquidityPoolData>(
-		QUERY_KEYS.LiquidityPools.sTSLA(walletAddress ?? '', network?.id!),
+		QUERY_KEYS.LiquidityPools.Balancer(walletAddress ?? '', synth, network?.id!),
 		async () => {
-			const {
-				contracts: { StakingRewardssTSLABalancer },
+			let {
```
Comment (dvd-schwrtz) [inline queries/liquidityPools/useBalancerPoolQuery.ts:41]: should still be const right?

**Pull request merged**